Makefile

```diff
@@ -125,6 +126,12 @@ kind-deploy-controller: install-crds
 	kubectl apply -f deploy/ -n $(CONTROLLER_NAMESPACE)
 	kubectl patch deployment $(IMG) -n $(CONTROLLER_NAMESPACE) -p "{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"$(IMG)\",\"env\":[{\"name\":\"WATCH_NAMESPACE\",\"value\":\"$(WATCH_NAMESPACE)\"}]}]}}}}"
 
+community-deploy-controller: install-crds
+	@echo installing $(IMG)
+	kubectl create ns open-cluster-management-agent-addon || true
+	kubectl -n open-cluster-management-agent-addon apply -f deploy/
+	kubectl -n open-cluster-management-agent-addon patch deployment $(IMG) -p "{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"$(IMG)\",\"env\":[{\"name\":\"WATCH_NAMESPACE\",\"value\":\"$(MANAGED_CLUSTER_NAME)\"}]}]}}}}"
+
```

Comment:
Let's move this new target up to the 'deploy' section at the top of the file.

Reply:
@ycao56 We probably need to do more consolidating in this file ourselves anyway, but based on https://github.com/open-cluster-management/governance-policy-framework/pull/100#discussion_r623079465, do you think this should also be something like:
```
deploy-controller: create-ns install-crds
	@echo installing $(IMG)
	kubectl -n $(CONTROLLER_NAMESPACE) apply -f deploy/
	kubectl set env deployment/$(IMG) -n $(CONTROLLER_NAMESPACE) WATCH_NAMESPACE=$(WATCH_NAMESPACE)
```

Reply:
done as suggested